README.md

```diff
@@ -1,179 +1,194 @@
+come from CentOS distributions, so if some packages names don't match
+on your distribution try to find their equivalent names): git, make,
+cmake, gcc-c++, python3.
 
-    git, make, python3, cmake, centreon-clib-devel
+For the projet compilation you need to have conan installed. Try to use
+the package manager given by your OS to install conan. ('apt' for
+Debian, 'rpm' for Red Hat, 'pacman' for Arch Linux, ...). It is prefered
+to install gcc before conan.
 
-If they are not installed, please install them.
-    $> apt install git make cmake python3
+Example:
 
-Then you will be able to resume the following steps :
-    $> git clone https://github.com/centreon/centreon-engine
-    $> cd centreon-engine
-    $> ./cmake.sh
+```shell
+_apt_ install conan
+```
 
-To speed up the compilation you can use make
-    $> cd build/ && make && make install
+If it does not work, conan can be installed with pip3:
 
-**For other distributions :**
-If you are on another distribution, then follow the steps below.
+```shell
+pip3 install conan
+```
 
-Check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+> All the dependencies pulled by conan are located in conanfile.txt. If
+> you want to use a dependency from your package manager instead of conan,
+> you need to remove it from conanfile.txt.
 
-    git, make, cmake, gcc-c++, python3, epel-release, centreon-clib-devel
+Then you have to add a remote conan repository, for that enter the command:
 
-For the projet compilation you need to have conan installed. Try to use the package manager given by your OS to install conan. ('apt' for Debian, 'rpm' for Red Hat, 'pacman' for Arch Linux, ...) It is prefered to install gcc before conan.
+```shell
+conan remote add centreon https://api.bintray.com/conan/centreon/centreon
+```
 
-Exemple :
-    $> _apt_ install conan
+Now, the list command should list two repositories:
 
-If it does not work, conan can be installed with pip3 :
-    $> pip3 install conan
+```shell
+conan remote list
+conan-center: https://conan.bintray.com [Verify SSL: True]
+centreon: https://api.bintray.com/conan/centreon/centreon [Verify SSL: True]
+```
 
-You can now compile centreon-engine :
-    $> git clone https://github.com/centreon/centreon-engine
-    $> mkdir -p centreon-engine/build
-    $> cd centreon-engine/build
-    $> conan install .. --build missing
+You can now compile Centreon Engine:
 
-Once the sources of Centreon Engine extracted go to the *./build/* directory and launch the CMake command.
-This will look for required dependencies and print a summary of the compilation parameters if everything went fine.
+```shell
+git clone https://github.com/centreon/centreon-engine
+mkdir -p centreon-engine/build
+cd centreon-engine/build
+conan install --remote centreon --build missing
```

Comment:
conan install --remote centreon --build missing **..**
The ".." is really important to specify the location of conanfile.txt